### DISCUSSION TITLE
请教怎么配置GFW模式？

### DISCUSSION THREAD

**Discussion by baodp** [Openwrt-Passwall/openwrt-passwall2#214] [Q&A] [answered]
听说可以配置GFW模式？
请教怎么配置？
谢谢。

**Comment by Travel2Here** [comment, marked as answer]
默认直连，proxy代理域名仅geosite:gfw即可

**Reply by baodp** [reply]
> geosite:gfw

谢谢搞定了。

**Comment by frankang** [comment]
但是在自动切换里面，有一个“如果主节点是分流”的选项，里面只能够应用于默认节点。如果我想用gfw模式，但同时希望能够使用自动切换功能，应该怎么设置？ @Travel2Here

**Comment by Deleted-Account** [comment]
默认订阅的 https://github.com/Loyalsoldier/v2ray-rules-dat
xiaorouji 给的 proxy 规则是 geosite:geolocation-!cn，已包含GFW，还包含一些墙内访问慢的国外网站

**Reply by NewyouthZS** [reply]
请问如何订阅呢？

**Reply by Deleted-Account** [reply]
Passwall2 的 Rule Manage 标签，勾选 geoip geosite 订阅就是了。
或者自己写脚本订阅其他版本的 geoip geosite，比如像我主要使用域名分流而不是IP分流，想使用精简版geoip，就可以用脚本更新只有百来KB大小的geoip。
如果用 sing-box 内核，geodata 貌似目前都需要自己写脚本订阅，和 xray 的不能混用。

geoip geosite 保存在
/usr/share/v2ray
/usr/share/singbox

**Reply by NewyouthZS** [reply]
多谢解惑，刚刚开始用PW2，发现分流策略要比PW精进了许多，确实更适合按域名分流。
